VATSpy.dat

```diff
@@ -18233,12 +18233,16 @@ UACN|Nur-Sultan||UACN
 UAII|Shymkent||UAII
 UATT|Aktyubinsk||UATT
 UBBA|Baku|UBBB|UBBA
-UBBA-S|Baku South|UBBA-S|UBBA-S
+UBBA-E|Baku East|UBBA_E|UBBA-E
+UBBA-S|Baku South|UBBA_S|UBBA-S
+UBBA-W|Baku West|UBBA_W|UBBA-W

```

Comment:
We can not accept these as per our policy:

> To keep the map clear and not too cluttered, any new additions to the dataset should cover whole FIR's (e.g. HECC/KZFW). Some exceptions can be made for further splits if the (sub)sector is controlled consistently outside of events and the amount of traffic justifies it. 

These are the UBBA hours this year:
![image](https://user-images.githubusercontent.com/35482228/200086366-e321c033-d62e-4cbd-8fc4-e0eb483fc3cf.png)

**(The only exception is UBBA-S due to its separation from the main UBBA FIR)**